elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -30,73 +14,66 @@ div.pl-order-blocks-bottom {
 }
 
 
-ul.dropzone {
-  /* This style applies to answer dropzone */
-  background-color: #FFA; /* Yellow */
-  min-height: 30px;
+ul.pl-order-blocks-connected-sortable li:hover{
+  cursor: move; /* For UX: The answer tiles shows a 'grab' cursor on mouse over */
+  outline: 2px solid blue;
+  outline-offset: -2px;
 }
 
-
-ul.pl-order-blocks-connected-sortable li, ul.dropzone li{
-  /* This style applies to the answer draggable tiles themselves */
-  background-color: #EFEFEF; /* Grey */
-  border: 2px solid #A9A9A9;
-  border-radius: 4px;
-  margin: 0 10px 10px 10px;
-  padding: 5px;
-  font-size: 1.2em;
-
-  display: block;
-
-  /* Show overflow text in the answer draggable tiles */
-  white-space: normal;
-  height: auto;
-  max-height: none;
-  overflow: visible;
+li.ui-state-highlight { 
+  /*This styling applies to the placeholder answer tile that appears when the user is dragging */
+  height: 3em !important; 
+  border: none !important;
+  list-style-type: none;
+  background-color: #90a4ae !important;
+  min-width: 3em !important;
 }
 
-ul.pl-order-blocks-connected-sortable li:hover, ul.dropzone li:hover{
-  cursor: move; /* For UX: The answer tiles shows a 'grab' cursor on mouse over */
+ul.header {
+  color: black;
+  list-style-type: none;
+  margin-bottom: 0px;
 }
 
+ul.inline li {
+  display: inline-block;
+  margin-right: 5px;
+}
 
-li.info {
-  /* This styling applies to the instruction text for the dropzones */
-  background-color: transparent !important;
-  border: none !important;
-  cursor: auto !important;
+ul.ui-sortable {
+  padding: 17px 17px 17px 17px;
+  height: 100%;
+  margin-bottom: 0px;
 }
 
-li.ui-state-highlight { 
-  /*This styling applies to the placeholder answer tile that appears when the user is dragging */
-  height: 2em !important; 
-  line-height: 1.5em !important;
-  border: none !important;
-  color: #efefff !important; 
-  background-color: #efefff !important; 
+ul.ui-sortable li {
+  margin-bottom: 5px;
+  border: 1px solid grey !important;
+  border-radius: 5px !important;
+  background-color: #f5f5f5;
+  font-size: 1.1em;
+  line-height: 1.1em;
```

Comment:
The line height is added because without it, the blocks looks not tall enough, and the font-size is increased slightly to slightly improve readability. I guess there might be a better way to do this?

Reply:
> the blocks looks not tall enough
I don't think so, I think they perfect without the size increase. Have you had students complain about the size? Or are there certain types of content that don't fit? 

I really think we shouldn't mess with the size here, just leave it default.

Reply:
Nah, I just liked how it looked with the increased line height :laughing:, I'll leave it default then, since I also saw how they looked without size increase when debugging CSS and they also look OK

Reply:
Wait, I found that if you disable the line-height, the element actually becomes bigger :neutral_face:, this might be a bootstrap thing

Reply:
just change it to 1em then I guess? That's what I did locally and it was what I actually expected it to look like.

Reply:
![image](https://user-images.githubusercontent.com/11191061/119707486-0d7d4600-be10-11eb-8b56-b3377d9c45e0.png)
When there are only one line for the block, `line-height: 1.1em` definitely seems overkill, but when you have multi-line text, it's barely adequate as the text starts to seem bunched

Reply:
did you also change the font-size back to 1em?

Reply:
Yes

Reply:
This is very odd, it look like this for me, which I don't see any issue with:
![image](https://user-images.githubusercontent.com/25016959/119709987-76ba8480-be23-11eb-8adf-0e613a633e01.png)